includes/Modules/Ads.php

```diff
@@ -173,9 +174,10 @@ public function get_debug_fields() {
 		$settings = $this->get_settings()->get();
 
 		return array(
-			'ads_conversion_id' => array(
-				'label' => __( 'Conversion Tracking ID', 'google-site-kit' ),
+			'ads_conversion_tracking_id' => array(
+				'label' => __( 'Ads Conversion Tracking ID', 'google-site-kit' ),
 				'value' => $settings['adsConversionID'],
+				'debug' => Debug_Data::redact_debug_value( $settings['adsConversionID'] ),
```

Comment:
This is how IDs are formatted in debug data (clipboard). See GA for comparison https://github.com/google/site-kit-wp/blob/029750d5e5394cefd9dac2a971fb51da76d90d28/includes/Modules/Analytics_4.php#L401